Dockerfile

```diff
@@ -3,17 +3,22 @@ MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
 COPY prometheus                             /bin/prometheus
 COPY promtool                               /bin/promtool
-COPY documentation/examples/prometheus.yml  /etc/prometheus/prometheus.yml
+COPY documentation/examples/prometheus.yml  /etc/prometheus/config/prometheus.yml
 COPY console_libraries/                     /usr/share/prometheus/console_libraries/
 COPY consoles/                              /usr/share/prometheus/consoles/
 
-RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/
+RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/ && \
+    addgroup -g 1000 prometheus && \
+    adduser -s /bin/sh -S -H -u 1000 -G prometheus prometheus && \
+    mkdir -p /prometheus && \
+    chown -R prometheus:prometheus /etc/prometheus /prometheus
 
 EXPOSE     9090
-VOLUME     [ "/prometheus" ]
+VOLUME     [ "/prometheus", "/etc/prometheus/config" ]
```

Comment:
why add the config dir as a volume? this can simply be mounted here at runtime if required. plus above comment about using /etc/prometheus directly rather than subdir